frontend/src/game/Cols.scss

```diff
@@ -20,4 +20,27 @@
       }
     }
   }
+
+  .helper {
+    position: fixed;
+    bottom: 0;
+    z-index: 101;
+
+    --card-width: 240px;
+    --card-height: 340px;
+
+    &.left {
+      left: 0;
+    }
+
+    &.right {
+      right: 0;
+    }
+
+    .card {
```

Comment:
We might not need the card container any more if you're using raw CardBase.
I think that component inherits the size from that css var --card-height etc.

I'm mainly keen to get things away from that global style.css wherever possible